### PR TITLE
Add tests for the fixed precision encoder

### PR DESCRIPTION
**Pull request by arturomf94**
## Description
This PR will add tests for the fixed precision encoder.

---

Closes: https://github.com/OpenMined/SyMPC/issues/1

---

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Comment by arturomf94** [comment]
Force-pushed to rebase with `main` branch.

**Comment by arturomf94** [comment]
There's a warning when encoding floats: 

```
"DeprecationWarning: an integer is required (got type float).  Implicit conversion to integers using __int__ is deprecated, and may be removed in a future version of Python."
```

**Comment by arturomf94** [comment]
Forced push to solve conflicts in `conftest.py`

File: tests/sympc/encoder/fp_encoder_test.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    fp_encoder = FixedPointEncoder()
+    # Test encoding with tensors.
+    tensor = torch.Tensor([1, 2, 3])
+    encoded_tensor = fp_encoder.encode(tensor)
+    target_tensor = torch.LongTensor([1, 2, 3]) * fp_encoder.scale
+    assert (encoded_tensor == target_tensor).all()
+    # Test encoding with foats.
+    test_float = 42.0
+    encoded_float = fp_encoder.encode(test_float)
+    target_float = torch.LongTensor([42.0]) * fp_encoder.scale
```
Comment (LaRiffle) [inline tests/sympc/encoder/fp_encoder_test.py:36]:
```suggestion
    target_float = torch.LongTensor([42]) * fp_encoder.scale
```

**Review by LaRiffle — APPROVED** [review]
Thanks! 🙌

**Pull request merged**